backtester/engine/backtest_test.go

```diff
@@ -1479,9 +1411,8 @@ func TestLiveLoop(t *testing.T) {
 	wg.Add(1)
 	go func() {
 		err = bt.liveCheck()
-		if !errors.Is(err, nil) {
-			t.Errorf("received '%v' expected '%v'", err, nil)
-		}
+		assert.NoError(t, err)
```

Comment:
out of scope: we shouldn't be calling these and below in a go func, it becomes unstable.